lib/bson/error/invalid_decimal128_argument.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module BSON
+  class Error
+    # Raised when trying to create a Decimal128 from an object that is neither a String nor a BigDecimal.
+    #
+    # @api private
```

Comment:
The constructors are private, but the classes shouldn't be. This exception is reported to applications right? Can't be private if yes.

Reply:
done